libs/cuvslam/test/tracker_test.cpp

```
@@ -110,6 +111,29 @@ TEST_F(TrackerTest, GtAlignModeRequiresManualDispatch) {
   EXPECT_THROW(Tracker(rig, cfg.odometry, &cfg.slam), std::invalid_argument);
 }
 
+TEST_F(TrackerTest, RejectsNonFiniteCalibration) {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)